nms/packages/magmalte/fbc_js_core/ui/host/Organizations.js

```diff
@@ -191,17 +206,17 @@ function Organizations(props: Props) {
   });
   useEffect(() => {
     if (organizations?.length) {
-      getUsers(organizations, setUsers);
+      getUsers(organizations, setUsers, enqueueSnackbar);
     }
-  }, [organizations, addingUserFor]);
+  }, [organizations, addingUserFor, enqueueSnackbar]);
 
   if (error || isLoading || !organizations) {
     return <LoadingFiller />;
   }
 
   const onDelete = org => {
     props
-      .confirm('Are you sure you want to delete this org?')
+      .confirm('Are you sure you want to delete this organization?')
       .then(async confirm => {
         if (!confirm) return;
         await axios.delete(`/host/organization/async/${org.id}`);
```

Comment:
Do I understand it correctly that one delete the `host` organization?

Reply:
It deletes the selected organization https://github.com/magma/magma/pull/12394/files#diff-b927e1ba7650dd78506eb4417cbb6821d2ff49e38e5f34e2a22dab7119999202R325

Reply:
Sorry, what I meant is that one can actually delete the `host` organisation and break everything. But that might be an off topic discussion here.

Reply:
yes it does delete the host org 😕

Reply:
I think this was something we didn't catch during design mock-ups. Let's make sure deleting the host organization is impossible.